com.wso2telco.dep.hub.smsmessagingapi/com.wso2telco.dep.hub.smsmessagingapi.synapse/src/main/synapse-config/api/smsmessagingAPI.xml

```diff
@@ -8,6 +8,22 @@
         <outSequence/>
         <faultSequence/>
     </resource>
+    <resource methods="POST" protocol="http" uri-template="/outbound/notification/{subscriptionID}">
+        <inSequence>
+            <!-- SMS : Delivery receipt notification (Posted to Notify URL) -->
+            <sequence key="com.wso2telco.dep.hub.smsmessagingapi.handle.outbound.notification.Sequence"/>
+        </inSequence>
+        <outSequence/>
+        <faultSequence/>
```

Comment:
Fault sequence is missing. I think you are waiting to add this at the last minute. I would add this at the creation of the resource. :)